xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/diff.css

```diff
@@ -55,6 +55,21 @@ td.diff-line {
   white-space: pre-wrap;
 }
 
+/*
+Prevent the code formatting for diff lines containing a decision form as it breaks the display of the form.
```

Comment:
Maybe we should use a common comment style. Above in the same file I see for ex:

```
/**
 * Raw (Wiki Syntax) Diff
 */
```

I don't know what's the coding style for CSS files though. Seems we're missing comments in https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/XhtmlCssCodeStyle/

Reply:
`/**` is used very rarely afaics and does not feel "css" to me. It seems to be used most often to delimit subsections of the file (@mflorea can you confirm?).

Reply:
`/*` directly followed by a line break is used ever less often apparently. I have the habit to use this form for multi line comments. I'll do a proposal.

Reply:
The proposal: https://forum.xwiki.org/t/css-comments-code-style/8266

Reply:
> It seems to be used most often to delimit subsections of the file

Yes.

Reply:
On my side the expected style for this comment is this https://github.com/xwiki/xwiki-platform/blob/xwiki-platform-13.0/xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/diff.css#L209 (below on the same CSS file).